packaging_automation/templates/citus-enterprise.spec.tmpl

```diff
@@ -0,0 +1,347 @@
+%global pgmajorversion 11
+%global pgpackageversion 11
+%global pginstdir /usr/pgsql-%{pgpackageversion}
+%global sname citus-enterprise
+%global pname citus
+%global debug_package %{nil}
+%global unencrypted_package "%{getenv:UNENCRYPTED_PACKAGE}"
+
+Summary:	PostgreSQL-based distributed RDBMS
+Name:		%{sname}%{?pkginfix}_%{pgmajorversion}
+Provides:	citus_%{pgmajorversion}
+Conflicts:	citus_%{pgmajorversion}
+Version:	{{rpm_version}}
+Release:	1%{dist}
+License:	Commercial
+Group:		Applications/Databases
+Source0:	https://github.com/citusdata/{{project_name}}/archive/v{{version}}.tar.gz
+URL:		https://github.com/citusdata/{{project_name}}
```

Comment:
I think we should not have `{{project_name}}` vars in template files, since we can anyway hardcode it.

Can you go over `.tmpl` files and replace occurrences of `{{project_name}}` with the actual project names ?

Reply:
Fixed